Xamarin.Essentials/FilePicker/FilePicker.tizen.cs

```diff
@@ -11,10 +11,12 @@ namespace Xamarin.Essentials
 {
     public static partial class FilePicker
     {
-        static Task<IEnumerable<FilePickerResult>> PlatformPickAsync(PickOptions options, bool allowMultiple = false)
+        static List<FilePickerResult> resultFiles = new List<FilePickerResult>();
```

Comment:
Why was this moved out of the scope? I also see another variable that is unused.

https://github.com/xamarin/Essentials/blob/f8045e0b835e8ce30eab712a9456a72161da6acb/Xamarin.Essentials/FilePicker/FilePicker.tizen.cs#L31

Does it not work to use that and then
`return await tcs.Task`

I also think what is happening is that files are being added to the list, but never cleared out...

Reply:
Thanks, @mattleibow and @redth! 😉